src/main/java/com/notably/logic/parser/NewCommandParser.java

```diff
@@ -39,11 +39,11 @@ public NewCommandParser(Model notablyModel) {
 
         if (!ParserUtil.arePrefixesPresent(argMultimap, PREFIX_TITLE)
                 || !argMultimap.getPreamble().isEmpty()) {
-            throw new ParseException(String.format("Invalid Command"));
+            throw new ParseException("Invalid Command");
```

Comment:
Do you feel we shd make our error message more user-friendly, since the user might not be able to understand what `command` is referring to? Maybe we can say something like `<user input invalid cmd> is not a valid command. To see the list of available commands, type: help`.

Reply:
Will do this in another PR :D